### ISSUE TITLE
require-param still broken

### ISSUE THREAD

**Issue by Announcement** [gajus/eslint-plugin-jsdoc#38]
consider the following code, documented according to the official [jsdoc specification](http://usejsdoc.org/tags-param.html)
~~~ es6
`/**`
 * Interception layer to revert back to transform if data is missing.
 *
 * @function callback
 *
 * @param {...Array} input - All of the variables to be accepted by method.
 *
 * @returns {Object} Output of the source curry method.
 **/
function callback (...input) {
  var response

  inputs = combine(params, array(arguments))
  context = update(this)
  response = fetch(inputs)

  return response
}
~~~
yet it still throws the error

Type | Message | Rule ID
-|-|-
warning │ Missing JSDoc @param "input" declaration. │ jsdoc/require-param

here is my  `.eslintrc.js` file
~~~ javascript
module.exports = {
  env: {
    browser: true,
    node: true,
    es6: true,
    mocha: true
  },

  extends: ['standard'],

  plugins: ['standard', 'promise', 'jsdoc'],

  rules: {
    'max-len': [2, { code: 80, ignoreComments: true }],
    'max-nested-callbacks': [2, { max: 3 }],
    'consistent-return': 2,
    'global-require': 2,

    'vars-on-top': 1,
    complexity: [1, { max: 2 }],
    'max-depth': [1, { max: 3 }],
    'max-params': [1, { max: 3 }],

    'jsdoc/check-param-names': 1,
    'jsdoc/check-tag-names': 1,
    'jsdoc/check-types': 1,
    'jsdoc/newline-after-description': 1,
    'jsdoc/require-description-complete-sentence': 1,
    'jsdoc/require-example': 0,
    'jsdoc/require-hyphen-before-param-description': 1,
    'jsdoc/require-param': 1,
    'jsdoc/require-param-description': 1,
    'jsdoc/require-param-type': 1,
    'jsdoc/require-returns-description': 1,
    'jsdoc/require-returns-type': 1
  },

  settings: {
    jsdoc: {
      tagNamePreference: {}
    }
  }
}
~~~

**Comment by devnev0** [comment]
I have the same problem. It' due to comment-parser 0.4.1 dependency. Deleting  '^' symbol on this line:
`"comment-parser": "^0.4.0",` 
solves the problem.

**Comment by gajus** [comment]
Did anyone raise an issue with comment-parser? If you have, please link it to this issue.

**Comment by devnev0** [comment]
it' the same things that here:
https://github.com/gajus/eslint-plugin-jsdoc/issues/41
isn't it?

**Comment by gajus** [comment]
Looks like it. Does it mean it is no longer an issue?

**Comment by brokentone** [comment]
@gajus I have confirmed that the last few days concern with comment-parser@0.4.1 was fixed with 0.4.2. I'm not sure if this is related to this original issue from June.

**Comment by gajus** [comment]
I have bumped the dependencies. Closing this on the assumption that the `comment-parser` have addressed the issue.
